README.md

```diff
@@ -103,7 +103,7 @@ Visit our [`styled-engine` guide](https://mui.com/material-ui/guides/styled-engi
 ### Diamond 💎
 
 <p align="center">
-  <a href="https://octopus.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="128" width="128" src="https://avatars3.githubusercontent.com/u/1287123?s=256" alt="octopus" title="Repeatable, reliable deployments" loading="lazy" /></a>
+  <a href="https://octopus.com/?utm_source=MUI&utm_medium=referral&utm_content=readme" rel="noopener sponsored" target="_blank"><img height="128" width="128" src="https://i.ibb.co/w0HF0Nz/Logo-Blue-140px-rgb.png" alt="octopus" title="Repeatable, reliable deployments" loading="lazy" /></a>
```

Comment:
What's the difference? The objective of using the GitHub user ID to fetch the logo was to not have to update it when they rebrand. Also, if we have a custom icons, why use https://imgbb.com/ and not our own website?

Reply:
The previous one didn't have a transparent background, so I went to Octopus' website, downloaded the logo, and uploaded it to this image hosting. But no big deal if we have to roll it back, it was bugging me a bit the white background when the docs are using dark mode.

Reply:
Ah OK, +1 to improve the dark mode experience. In this case, I think that if we could host the image alongside the other sponsor images, it would be great.

Reply:
Where do we host the image ourselves?

Reply:
Let's improve this as a next step, we can probably have all sponsor images under `docs/public/static/sponsors`.

Reply:
But the resolution is wrong, it's 140 pixels, it should be 256 pixels